packages/client/src/client.js

```diff
@@ -42,19 +42,22 @@ export class PercyClient {
   env = new PercyEnv(process.env);
   clientInfo = new Set();
   environmentInfo = new Set();
+  testing = false;
 
   constructor({
     // read or write token, defaults to PERCY_TOKEN environment variable
     token,
     // initial user agent info
     clientInfo,
     environmentInfo,
+    testing,
     // versioned api url
     apiUrl = PERCY_CLIENT_API_URL
   } = {}) {
     Object.assign(this, { token, apiUrl });
     this.addClientInfo(clientInfo);
     this.addEnvironmentInfo(environmentInfo);
+    this.testing = testing || false;
```

Comment:
* this won't be required, below comment is resolved ( code moved to `core` package)